### ISSUE TITLE
Add a search bar to the projects list

### ISSUE THREAD

**Issue by 3kh0** [3kh0/3kh0.github.io#67]
### What is this request?

New feature

### Please describe the new feature here

Add a search bar to the projects list

**Comment by nicolasog12** [comment]
To add a search bar, the /projects/index.html shouln't be blank...

**Comment by 3kh0** [comment]
Yeah, when you type, the projects get filtered, like a built-in ctrl+f

**Comment by nicolasog12** [comment]
This might help: https://www.w3schools.com/howto/howto_js_search_menu.asp

**Comment by 3kh0** [comment]
@nicolasog12 Thanks for the link, I will try to get this done soon!

**Comment by 3kh0** [comment]
Welp I did it first try, pog!

**Comment by nicolasog12** [comment]
#poggers

**Comment by Proph3t-3036** [comment]
#poggers!

**Comment by RuralAnemone** [comment]
e🅱ic poggers

**Comment by 3kh0** [comment]
![aZr9p4p_460s](https://user-images.githubusercontent.com/58097612/193666435-f8704428-d9e7-48c0-8373-3c33be5db865.jpg)

**Comment by RuralAnemone** [comment]
🐇🥞  &nbsp; &nbsp; &nbsp; &nbsp; 🔫🐃
